app/components/live-participants.js

```diff
@@ -8,4 +8,16 @@ export default class LiveParticipantsComponent extends Component {
   get liveService() {
     return getOwner(this).lookup('service:live');
   }
+
+  get participantsCount() {
+    let count = 0;
+    let peers = this.args.peers;
+    let role = this.args.role;
```

Comment:
Nit: Does these possible
`let { peers,  role} = this.args`

Reply:
done